src/tools/gophish_export.py

```diff
@@ -253,6 +253,43 @@ def get_application(rawEvent):
     return application
 
 
+def export_user_reports(api, assessment_id):
+    """Build and export a user_report JSON object for each campaign in an assessment."""
+    campaign_ids = get_campaign_ids(api, assessment_id)
+
+    for campaign_id in campaign_ids:
+        first_report = None
+        user_report_doc = dict()
+        campaign = get_campaign_data(api, campaign_id)
+
+        # iterate over clicks and find the earliest click
+        for click in campaign["clicks"]:
+            click_time = datetime.strptime(
+                click["time"].split(".")[0], "%Y-%m-%dT%H:%M:%S"
+            )
+            if first_report is None or click_time < first_report:
+                first_report = click_time
+
+        user_report_doc["customer"] = None
```

Comment:
Is a `null` the desired output in the JSON for this field? Your README change indicates the field is a `string`. A `null` and a `string` are not the same, so I am suggesting we instead output an empty string for this field.

I have also added a comment to provide some insight on why such a field is in the output in the first place. @dav3r I would appreciate your input on making this comment better.

```suggestion
        # The "customer" field is a placeholder added for operator convenience when
        # working with the JSON file created.
        user_report_doc["customer"] = ""
```

Reply:
@mcdonnnj - I like the comment.  I'd only make a minor change to minimize any confusion:
```
# The "customer" field is a placeholder added for operator convenience when
# working with the JSON file created.
```

Reply:
This is a good point to clarify.  Thanks for pointing this out @mcdonnnj.

Reply:
This is changed in 3c54a97003c86809fdefc23a46a7ca562492e01c